x-pack/plugins/ml/public/application/explorer/_explorer.scss

```diff
@@ -219,9 +219,12 @@
     height: 30px;
     border-bottom: 0px;
     border-radius: 2px;
```

Comment:
We can use `$euiBorderRadius / 2` to get to this value.

Reply:
Changed in https://github.com/elastic/kibana/pull/70063/commits